<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>697</height>
   </rect>
  </property>
  <widget class="QOsgViewer" name="centralWidget">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>1</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>901</width>
     <height>21</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="canvasMenu">
    <property name="title">
     <string>&amp;Canvas</string>
    </property>
    <addaction name="cubeMappedHimmelAction"/>
    <addaction name="polarMappedHimmelAction"/>
    <addaction name="paraboloidMappedHimmelAction"/>
    <addaction name="sphereMappedHimmelAction"/>
    <addaction name="separator"/>
    <addaction name="proceduralHimmelAction"/>
    <addaction name="proceduralHimmelWithCloudsAction"/>
    <addaction name="separator"/>
    <addaction name="insert3DObjectFromFileAction"/>
    <addaction name="clearSceneAction"/>
    <addaction name="separator"/>
    <addaction name="quitAction"/>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="aboutAction"/>
   </widget>
   <widget class="QMenu" name="menu_Debug">
    <property name="title">
     <string>&amp;Debug</string>
    </property>
    <addaction name="moonLockAction"/>
    <addaction name="sunLockAction"/>
    <addaction name="separator"/>
    <addaction name="textOverlayAction"/>
   </widget>
   <addaction name="canvasMenu"/>
   <addaction name="menu_Debug"/>
   <addaction name="helpMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="openAction">
   <property name="text">
    <string>&amp;Import...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="paintAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Paint</string>
   </property>
   <property name="shortcut">
    <string>B</string>
   </property>
  </action>
  <action name="panAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>P&amp;an</string>
   </property>
   <property name="shortcut">
    <string>H</string>
   </property>
  </action>
  <action name="zoomInAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Zoom &amp;In</string>
   </property>
   <property name="shortcut">
    <string>Z</string>
   </property>
  </action>
  <action name="zoomOutAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Zoom &amp;Out</string>
   </property>
   <property name="shortcut">
    <string>Shift+Z</string>
   </property>
  </action>
  <action name="actualPixelsAction">
   <property name="text">
    <string>&amp;Actual Pixels</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
  <action name="preferencesAction">
   <property name="text">
    <string>&amp;Preferences...</string>
   </property>
  </action>
  <action name="quitAction">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAf">
   <property name="text">
    <string>af</string>
   </property>
  </action>
  <action name="loadLayoutAction">
   <property name="text">
    <string>&amp;Load Layout...</string>
   </property>
  </action>
  <action name="saveCurrentLayoutAction">
   <property name="text">
    <string>&amp;Save Current Layout</string>
   </property>
  </action>
  <action name="saveCurrentLayoutAsAction">
   <property name="text">
    <string>Save Current Layout &amp;As...</string>
   </property>
  </action>
  <action name="closeAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="exportAction">
   <property name="text">
    <string>&amp;Export...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="eraseAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Erase</string>
   </property>
   <property name="shortcut">
    <string>E</string>
   </property>
  </action>
  <action name="originalViewAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;Original</string>
   </property>
   <property name="shortcut">
    <string>O</string>
   </property>
  </action>
  <action name="resultViewAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;Result || Debug</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="inputViewAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;Input Mask</string>
   </property>
   <property name="shortcut">
    <string>I</string>
   </property>
  </action>
  <action name="layerViewAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Show &amp;Supportive Layer</string>
   </property>
  </action>
  <action name="actionDf">
   <property name="text">
    <string>df</string>
   </property>
  </action>
  <action name="createNewBrushAction">
   <property name="text">
    <string>&amp;Create new Brush...</string>
   </property>
  </action>
  <action name="actionAsd">
   <property name="text">
    <string>asd</string>
   </property>
  </action>
  <action name="cubeMappedHimmelAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;CubeMapped Himmel</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
  <action name="polarMappedHimmelAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;PolarMapped Himmel</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+P</string>
   </property>
  </action>
  <action name="proceduralHimmelAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Procedural Himmel</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+P</string>
   </property>
  </action>
  <action name="insert3DObjectFromFileAction">
   <property name="text">
    <string>&amp;Insert 3D-Objects from File ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="clearSceneAction">
   <property name="text">
    <string>Clea&amp;r Scene</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="paraboloidMappedHimmelAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Para&amp;boloidMapped Himmel</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+B</string>
   </property>
  </action>
  <action name="sphereMappedHimmelAction">
   <property name="text">
    <string>&amp;Sphere-Mapped Himmel</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="moonLockAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Camera Lock on &amp;Moon</string>
   </property>
   <property name="shortcut">
    <string>Shift+M</string>
   </property>
  </action>
  <action name="sunLockAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Camera Lock on &amp;Sun</string>
   </property>
   <property name="shortcut">
    <string>Shift+S</string>
   </property>
  </action>
  <action name="textOverlayAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Show Text Overlay</string>
   </property>
  </action>
  <action name="proceduralHimmelWithCloudsAction">
   <property name="text">
    <string>Procedural Himmel with Clouds</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QOsgViewer</class>
   <extends>QWidget</extends>
   <header>gui/qosgviewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
